File: VB_ScratchPad/frmMain.vb
Public Class frmMain
    Private Sub btnSayHello_Click(sender As Object, e As EventArgs) Handles btnSayHello.Click
        lblSayHello.Text = "Hello World!"
        lblSayName.Text = "My name is James!!!"
    End Sub

    Private Sub btnGoodbye_Click(sender As Object, e As EventArgs) Handles btnGoodbye.Click
        Application.Exit()
    End Sub
End Class

File: CS_ScratchPad/frmMain.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CS_ScratchPad
{
    public partial class frmMain : Form
    {
        public frmMain()
        {
            InitializeComponent();
        }

        private void btnSayHello_Click(object sender, EventArgs e)
        {
            lblSayHello.Text = "Hello World!";
            lblSayName.Text = "My name is James!!!";
        }

        private void btnGoodbye_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }
    }
}
